app/models/webhook_delivery.rb

```diff
@@ -1,5 +1,8 @@
 class WebhookDelivery < ApplicationRecord
   belongs_to :course
 
-  enum event: { submission_created: "submission.created" }
+  enum event: {
+    submission_created: "submission.created",
+    student_added:      "student.added",
```

Comment:
```suggestion
    student_created:      "student.created",
```

@mpraglowski: We feel it's good to have a naming schema like this, just to align with the terminologies we use for basic CRUD operations: created, updated, deleted, etc. Basically, to think more from the database operation point of view.

Reply:
I've blogged some time ago about domain events - see Rule 1 from https://blog.arkency.com/2016/05/the-anatomy-of-domain-event/ :) 

From the business point of view `student` is never created. He/she is registered/enrolled etc. We do not create students. I would prefer to seek added (as this is the terminology used in the PF system).

> Basically, to think more from the database operation point of view.
That's the point here - we should code from business point of view ... not the database. Database could handle any string, they do not care ;) Business people usually do - and it is a good thing to speak theirs language.

This operation on UI is named: "ADD NEW STUDENTS" - that's why `student_added` otherwise I would use registered or enrolled ;)

Reply:
That sounds good. Will retain as it is and merge :)